vizro-core/src/vizro/models/_dashboard.py

```diff
@@ -67,9 +67,13 @@ def pre_build(self):
         # For now the homepage (path /) corresponds to self.pages[0].
         # Note redirect_from=["/"] doesn't work and so the / route must be defined separately.
         for order, page in enumerate(self.pages):
-            path = page.path if order else "/"
             dash.register_page(
-                module=page.id, name=page.title, path=path, order=order, layout=partial(self._make_page_layout, page)
+                module=page.id,
+                name=page.title,
+                title=f"{self.title}: {page.title}" if self.title else page.title,
```

Comment:
This one line is the only functional change in the PR.

Reply:
I guess if people do want to overwrite it, they would overwrite the pre-build of the model here?

I was actually wondering how this works compared to the `dash.app.title`. If you set the `dash.app.title`, I noticed that it appears for a second and then it gets overwritten by anything defined in `dash.register_page`.

This might be worth mentioning in the docs if people try to overwrite it via the `dash.app.title`. This wouldn't actually work, because in the end it gets overwritten by the code here again. But I think this is also how it works in a pure Dash app?

```
app = Vizro(assets_folder="../assets").build(dashboard)
app.title = "My title"
app.run()
```

Reply:
This is indeed how it works in a pure Dash app also:
1. `Dash(title="X")` corresponds to the property `dash.app.title` and sets the initial value of `<title>X</title>`
2. but this only appears very briefly when you're using Dash pages, because it gets [overwritten straight away](https://github.com/plotly/dash/blob/7527383050b6fe59a11833bbeac6f1a741436084/dash/dash.py#L2159) by  `dash.page_registry[page.id]["title"]`
3. after this, the original app `title=X` has no effect on anything

So for Vizro users where Dash pages is always used, the `title` argument basically has no effect. @AnnMarieW please do correct me if I've missed something here.

We have possibly added to the confusion here because we have our own `Dashboard.title` property, and it's not obvious to a user what effect that has on anything. I considered before whether we should do `dash.app.title = dashboard.title` in `Vizro.build()` but decided it was basically pointless because, like you say, it just gets overwritten straight away. I think it would be a very small improvement though, since that way you wouldn't see the "Dash" title at all, so let me add it in here.

Reply:
Change made in [c00fe4e](https://github.com/mckinsey/vizro/pull/228/commits/c00fe4ed0b58ab9b13645ab4aeb8484b08dde524).